src/gui/askexperimentalvirtualfilesfeaturemessagebox.h

```diff
@@ -11,7 +11,7 @@ namespace OCC {
 class AskExperimentalVirtualFilesFeatureMessageBox : public QMessageBox
 {
 public:
-    explicit AskExperimentalVirtualFilesFeatureMessageBox(QWidget *parent = nullptr);
+    AskExperimentalVirtualFilesFeatureMessageBox(QWidget *parent = nullptr);
 };
```

Comment:
why?

Reply:
`QMainWindow *` is not implicitly casted to `QWidget *` otherwise, and requires a `static_cast` to work.